### PR TITLE
add tips & suggestions for hamburg

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

File: contents/news/hamburg-sightseeing-and-tips/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+First time visiting Hamburg? Having some days left before or after JS Unconf?
+
+[Read our tips and suggestions!](http://2014.jsunconf.eu/news/hamburg-sightseeing-and-tips/)
+
+
+## Public Transportation
+
+The public transportation in Hamburg is done by the [HVV](http://www.hvv.de/en/index.php). The important thing to know is that you **need to buy a ticket before you go on the platform** and you don't need to clock in the tickets, they are valid after you bought them. Don't go behind the golden lines in the ground without a ticket! For busses you can buy your ticket at the driver.
+
+A "Tagesticket" - a ticket that's valid the whole day and large parts of Hamburg is the best option most of the time, even if you just take 2 rides. It costs around 7€.
```
Comment (ro-ka) [inline contents/news/hamburg-sightseeing-and-tips/index.md]: … and _in_ large parts of Hamburg …

File: contents/news/hamburg-sightseeing-and-tips/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Sightseeing
+
+Hamburg has one of the busiest ports in europe (third after Antwerp and Rotterdam). At the subway station "Landungsbrücken" harbor tours are starting, where you will get some informations regarding the harbor while driving around with a small boat. Protip: There are also HVV ferrys starting there, which you can use with a "Tagesticket". With one of these ferries you can also drive to the station "Övelgönne" - where you can take a walk at the small beach of the Elbe river.
+
+On Sunday morning there is the famous Hamburger Fischmarkt, which is also a popular destination for people on their way back home after a party night, e.g. after visiting the bars and clubs of the famous Reeperbahn.
```
Comment (ro-ka) [inline contents/news/hamburg-sightseeing-and-tips/index.md]: On sunday morning, _starting at 5 a.m.,_ there is the famous …

File: contents/news/hamburg-sightseeing-and-tips/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+The [Speicherstadt](http://en.wikipedia.org/wiki/Speicherstadt) is the old warehouse district of Hamburg. Even today the old warehouses are still used for goods like cocoa, coffee or carpets. But some of the buildings are hosting attractions for visitors, e.g. [The Hamburg Dungeon](http://en.wikipedia.org/wiki/Hamburg_Dungeon) and the [Miniatur Wunderland](http://en.wikipedia.org/wiki/Miniatur_Wunderland).
+
+[Planten un Bloomen](http://en.wikipedia.org/wiki/Planten_un_Blomen) in the central of Hamburg near the subwaystation Dammtor is a nice park for having a rest in the sun. In the park there is also the tropical house of the University Hamburg, the entrance is free.
```
Comment (ro-ka) [inline contents/news/hamburg-sightseeing-and-tips/index.md]: … in the _center_ of Hamburg …

File: contents/news/hamburg-sightseeing-and-tips/index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+From the top of [St. Michael's Church](http://en.wikipedia.org/wiki/St._Michael's_Church,_Hamburg) you can see all parts of Hamburg. It costs some money to get up to the tower and currently the lift is not working, and you have to take the stairs, but the sight is awesome.
```
Comment (ro-ka) [inline contents/news/hamburg-sightseeing-and-tips/index.md]: … lift is not working _so_ you have to take the stairs …

File: contents/news/hamburg-sightseeing-and-tips/index.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Depending on how long you are staying in Hamburg the [Harbor Birthday](http://www.hamburg.de/hafengeburtstag-english/) would be a fest you can vist, too.
```
Comment (ro-ka) [inline contents/news/hamburg-sightseeing-and-tips/index.md]: … staying in Hamburg_,_ the …

**Comment by ro-ka** [comment]
Nice writeup! :)

**Comment by robertkowalski** [comment]
thanks for the review. updated everything. good catches.

**Pull request merged**